src/components/CustomModal,/CustomModal.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {
+  Button,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Text
+} from '@chakra-ui/react';
+import i18next from 'i18next';
+
+export const CustomModal = ({ isOpen, onClose, onSubmit, title, text }) => {
```

Comment:
if you're making a custom modal for the whole app, you need to rename it and make it ore abstract. However your task was to create a specific modal for a specific purpose, so what you can do is rename this to DeleteComfirmationModal and then it works

Reply:
fixed